plugins/python/_hexchat.py

```diff
@@ -220,11 +219,10 @@ def ptr_getter(field):
     return ret
 
 
-# TODO: 'command' here shadows command above, and should be renamed to cmd
-def hook_command(command, callback, userdata=None, priority=PRI_NORM, help=None):
```

Comment:
I appreciate handling the TODOs but in Python these are kind of public API and can't be changed.

Reply:
Lets just keep this MR entirely to the win32 build changes.

Reply:
Sorry for the (very) long delay in response. 😅 
Removed all TODOs related code in this MR.